package.json

```diff
@@ -3,9 +3,20 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "fedx-scripts jest --coverage",
+    "lint": "fedx-scripts eslint --ext .js .",
+    "lint:fix": "fedx-scripts eslint --ext .js . --fix"
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "webpack-merge": "^5.10.0"
+  },
+  "devDependencies": {
+    "@commitlint/config-conventional": "^17.4.0",
+    "@edx/stylelint-config-edx": "^2.3.3",
+    "@openedx/frontend-build": "13.0.27",
```

Comment:
We must use the appropriate version of `frontend-build` for a particular release, since during deployment we will end up breaking all MFEs.

Reply:
The version we are currently using is the same version we are using in [our header](https://gitlab.raccoongang.com/owlox-team/productsforge/oex/frontend-component-header/-/blob/redwood-rg/package.json#L44). Which version do you mean?

Reply:
Yes, you are right